aegis_gym/sim/genesis/scene_director_genesis.py

```diff
@@ -42,9 +42,11 @@ def __init__(
         device: str = "cuda",
         show_render: bool = True,
         cfg: dict = SIM_CFG,
+        enable_scene_camera: bool = False,
     ):
         super().__init__(device, show_render)
         self.cfg = cfg
+        self.enable_scene_camera = enable_scene_camera
```

Comment:
To address previous comment:
```suggestion
        self.enable_scene_camera = enable_scene_camera
        self.camera = None
```